index.html

```diff
@@ -264,6 +264,28 @@ <h2 class="header-text">About <br>Myself</h2>
             </div>
           </div>
         </section>
+        <section class="contact-form"> 
+          <div>
+            <h2 class="contactme">Contact Me</h2>
+            <p class="c-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="https://formspree.io/f/xnqynezq" method="post">
+            <label for="Name"></label>
+            <input type="text" class="name-label" id="name-label" maxlength="30" placeholder="Name" required name="name">
+
+            <label for="Email"></label>
+            <input type="email" class="email-label" id="email-label" placeholder="Email" required name="Email">
+
+            <label for="Message"></label>
+            <textarea type="message" class="message-label" id="message-label" maxlength="500" placeholder="Write your message here" required name="message"></textarea>
+             
+
+
+
+          </form>
+          <button class="gtn-button" type="submit">Get in touch</button>
```

Comment:
- [ ] Your `Get in touch` button is not inside the form. As a result the form is not working as expected. Kindly move the button inside the form.